docs/krel/gcbmgr.md

```diff
@@ -0,0 +1,183 @@
+
+`gcbmgr` is a subcommand of `krel` for Release Managers to submit Kubernetes staging and release jobs to Google Cloud Build (GCB).
+
+`krel gcbmgr` is a wrapper around the `gcloud builds submit` command which passes a set of GCB substitutions (variables) to a GCB configuration for usage in [`anago`](/anago).
+
+**NOTE: `krel gcbmgr` is currently in development and its design may rapidly change. If you encounter errors, please file an issue in this repo.**
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Important notes](#important-notes)
+- [Alpha](#alpha)
+  - [Alpha Stage](#alpha-stage)
+  - [Alpha Release](#alpha-release)
+- [Beta](#beta)
+  - [Beta Stage](#beta-stage)
+    - [Branch cut (`x.y.0-beta.0`)](#branch-cut-xy0-beta0)
+    - [Post-branch cut (`x.y.0-beta.1` and beyond)](#post-branch-cut-xy0-beta1-and-beyond)
+  - [Beta Release](#beta-release)
+- [Release Candidate](#release-candidate)
+  - [Release Candidate (RC) Stage](#release-candidate-rc-stage)
+  - [Release Candidate (RC) Release](#release-candidate-rc-release)
+- [Official](#official)
+  - [Official Stage](#official-stage)
+  - [Official Release](#official-release)
+
+## Installation
+
+From this root of this repo:
+
+```shell
+./compile-release-tools krel
+```
+
+## Usage
+
+`krel gcbmgr [flags]`
+
+```shell
+Flags:
+      --branch string          Branch to run the specified GCB run against
+      --build-version string   Build version
+      --gcb-config string      If provided, this will be used as the name of the Google Cloud Build config file. (default "cloudbuild.yaml")
+      --gcp-user string        If provided, this will be used as the GCP_USER_TAG.
+  -h, --help                   help for gcbmgr
+      --project string         Branch to run the specified GCB run against (default "kubernetes-release-test")
+      --release                Submit a release run to GCB
+      --stage                  Submit a stage run to GCB
+      --stream                 If specified, GCB will run synchronously, tailing its' logs to stdout
+      --type string            Release type (must be one of: 'prerelease', 'rc', 'official') (default "prerelease")
+
+Global Flags:
+      --cleanup            cleanup flag
+      --log-level string   the logging verbosity, either 'panic', 'fatal', 'error', 'warn', 'warning', 'info', 'debug' or 'trace' (default "info")
+      --nomock             nomock flag
+      --repo string        the local path to the repository to be used (default "/tmp/k8s")
+```
+
+## Important notes
+
+- Default executions of `krel gcbmgr` run in mock mode. To run an actual stage or release, you **MUST** provide the `--nomock` flag.
+- Always execute the release process in the following order:
+  - mock stage
+  - mock release
+  - nomock stage
+  - nomock release
+- For staging jobs, it's fine to infer the build version using the version marker URLs.
+
+  Example:
+
+  ```shell
+  --build-version $(curl -Ls https://dl.k8s.io/ci/latest.txt)
+  ```
```

Comment:
Setting the build version should not needed any more, right:

https://github.com/kubernetes/release/blob/6d7bcdd636133b119e887fd314d0b84eee5b6414/cmd/krel/cmd/gcbmgr.go#L328-L335
https://github.com/kubernetes/release/blob/6d7bcdd636133b119e887fd314d0b84eee5b6414/pkg/release/release.go#L195-L217